### DISCUSSION TITLE
Is it better to use refiner for better realistic results ?

### DISCUSSION THREAD

**Discussion by Amit30swgoh** [lllyasviel/Fooocus#2680] [Q&A]
If yes then which best samler steps cfg and refiner time like 0.4 0.5 0.7 etc
ty!
was thinking of useing juggernet 9 v2 tun diffusion 30-40 steps as the refiner

**Comment by mashb1t** [comment]
this is completely up to you and your taste, feel free to experiment and maybe share your experiences.

**Comment by Rogala** [comment]
I use as a refiner [v60B1VAE](https://civitai.com/models/4201?modelVersionId=245598) with Refiner Switch At 0.4 since it is SD 1.5.
No two people can see the same, adjust to suit yourself.
